x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/data.tsx

```diff
@@ -8,10 +8,9 @@ import styled from 'styled-components';
 import { EuiHealth } from '@elastic/eui';
 import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
 import React from 'react';
+import { SeverityValue } from '../../../pages/detection_engine/rules/types';
```

Comment:
Would it be possible to start using the `common` types here?

e.g. 

https://github.com/spong/kibana/blob/d2156b7285c0397fe93a12cd8f1464ac430bc0e8/x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts#L256-L257